### ISSUE TITLE
model forward input should be more flexible

### ISSUE THREAD

**Issue by Yumin-Sun-00** [TylerYep/torchinfo#30]
I have such forward input arguments: forward(self, dict, tensor)
```
b = 16
summary(model, input_data=[{'512': torch.randn(b,1,256,256)}, torch.randn(b,1,256,256).long()])
```
Could you please add this feature in the future?

**Comment by TylerYep** [comment]
Handling more complex input types is definitely on the roadmap. Can you post the current error you are getting with this code?

If you want the current version of torchinfo to work, you can try passing the second parameter as a kwarg or using input_size instead.

**Comment by TylerYep** [comment]
Hi everyone, I recently published version 1.5.2, which should resolve all of the issues passing in a dictionary as the input.

`input_data` accepts whatever arguments your forward() function takes, so you can pass them in using a list, or as a kwargs dictionary:
```
tensors = {...}
summary(model, input_data=[tensors, other_args, ...])
```

Please open a new issue if you run into any problems with the new version!
